tests/rootless.sh

```diff
@@ -19,12 +19,16 @@
 # a new feature, please match the existing style. Add an entry to $ALL_FEATURES,
 # and add an enable_* and disable_* hook.
 
+set -e -u -o pipefail
+[ -v RUNC_USE_SYSTEMD ] || RUNC_USE_SYSTEMD=
```

Comment:
nit: `: ${RUNC_USE_SYSTEMD:=}` is slightly shorter

Reply:
Fixed